code.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "repodive-tools",
+  "description": {
+    "en": {
+      "shortDescription": "Scripts and tools to gather information about git repositories.",
+      "longDescription": "This repository is a collection of scripts and tools for a given repodiving effort. Repodiving in this context means going through a git repository and gathering relevant information for a specific purpose."
+    }
+  },
+  "status": "archival",
+  "permissions": {
+    "licenses": [
+      {
+        "URL": "LICENSE",
+        "name": "CC0-1.0"
+      }
+    ]
+  },
+  "organization": "Centers for Medicare & Medicaid Services",
+  "repositoryURL": "https://github.com/DSACMS/repodive-tools",
+  "vcs": "git",
+  "laborHours": null,
+  "platforms": [
+    "mac"
```

Comment:
In theory, it should be able to run on all platforms.

Sachin thinks that this is related to not being able to do 'multi-select' on cookiecutter. Perhaps we can get around this by asking about each platform separately, and then appending them to the field? I don't love that, but it might be a good work around?

Reply:
wrote this down and will coordinate with Nat for which direction we should take